### ISSUE TITLE
Where is `roles/plone.plone_server/tasks/main.yml`?

### ISSUE THREAD

**Issue by stevepiercy** [plone/ansible.plone_server#135]
When running through the tutorial, I kept hitting the problem fixed in #134. I had to manually edit `plone/ansible-playbook/playbook.yml:13` to the following:

`when: ansible_version.full is version('2.5.0', '<')`

Then after running the playbook, I had to add another instance of the same issue in `plone/ansible-playbook/roles/plone.plone_server/tasks/main.yml:5`.

Two questions:

1. Why did the change in #134 not get used when I ran the playbook?
2. Where the heck is `plone/ansible-playbook/roles/plone.plone_server/tasks/main.yml` in any of the repositories, or is it generated automatically somehow?

**Comment by smcmahon** [comment]
https://github.com/plone/ansible.plone_server/blob/master/tasks/main.yml

In a typical install, this has been picked up by running:

ansible-galaxy -p roles -r requirements.yml install 

The reason it's separately packaged is to allow folks who need more flexibility than the complete playbook provides, but would like the convenience of a role for the plone server alone.

It's entirely possible for the plone_server role to be out of sync with the playbook if a user has updated one without updating the other. It would be nice to have the playbook check to make sure there's a version match.

**Comment by stevepiercy** [comment]
OK, looks like a [patch is needed in the tutorial to append ` install`](https://github.com/plone/training/blame/master/deployment/ansible.rst#L79), correct? I can make a PR, if so.

**Comment by stevepiercy** [comment]
Correction, move `install` to the end of the command? That would then synch up with the README, too.

**Comment by stevepiercy** [comment]
After reading the docs for `ansible-galaxy`, I guess the order of the args in the command does not matter after all.

That aside, I think I found why. The playbook picks up the latest release v1.3.7 as specified in `requirements.yml`, but [PR #134 was merged after the last release](https://github.com/plone/ansible.plone_server/compare/1.3.7...master). It looks like we need to cut another release with the change.
